### ISSUE TITLE
Updater expects NINTENDO_SESSION_IDs for all three regions

### ISSUE THREAD

**Issue by ticky** [misenhower/splatoon2.ink#15]
Took me a bit to figure out why I was getting huge error logs. Turns out it’s because I only supplied an NA session ID. Might be worth it to add skip logic if the token is missing! :)

**Comment by misenhower** [comment]
That is true, in general there are a lot of things that could be cleaned up or improved with the updaters. For now you can just use the same session id for each region, the only issue will be that some languages won’t update properly and you’ll miss the international Splatfests.
